src/digital_identity_service/index.js

```diff
@@ -267,80 +261,31 @@ class DigitalIdentityService {
    * @param {string} receiptId
    */
   async fetchAndDecryptReceipt(receiptId) {
-    const receipt = await this.fetchEncryptedReceipt(receiptId);
+    const receiptResponse = await this.fetchEncryptedReceipt(receiptId);
```

Comment:
minor:

I think we had discussed renaming some of these methods

`fetchEncryptedReceipt` -> `fetchReceipt` (since the whole response is not actually encrypted, but just the profile(s)`

Reply:
Yup, true, will look into doing that.